src/constants.zig

```diff
@@ -53,9 +53,9 @@ comptime {
 
 /// The checkpoint interval is chosen to be the highest possible value that satisfies the
 /// constraints described below.
-pub const vsr_checkpoint_interval = journal_slot_count -
-    lsm_batch_multiple -
-    lsm_batch_multiple * stdx.div_ceil(pipeline_prepare_queue_max * 2, lsm_batch_multiple);
+pub const var_checkpoint_ops = journal_slot_count -
```

Comment:
**vsr**_checkpoint_ops, not **var**_checkpoint_ops

Reply:
Oh dear, that’s an excellent catch! 

I so clearly remember re-reading my sed invocation to make sure there are no typos 🤣